app/scripts/modules/amazon/src/loadBalancer/configure/common/targetGroupValidators.ts

```diff
@@ -0,0 +1,58 @@
+import { get } from 'lodash';
```

Comment:
Awesome work with the validators.  The factory functions can be typed to return `IValidator`